src/register/register.ts

```diff
@@ -30,9 +31,16 @@ export class Register {
    */
   private static registers: { [key: string]: IRegisterContent } = {
     '"': { text: "", registerMode: RegisterMode.CharacterWise },
-    '*': { text: "", registerMode: RegisterMode.CharacterWise }
+    '*': { text: "", registerMode: RegisterMode.CharacterWise },
+    '+': { text: "", registerMode: RegisterMode.CharacterWise }
   };
 
+  private static systemClipboardRegisters: string[] = ["*", "+"];
+
+  public static isSystemClipboardRegister(register: string): boolean {
```

Comment:
Instead of this, make IRegisterContent have another optional field called `isClipboardRegister` and put the value on there.